core/src/main/java/discord4j/core/object/entity/Member.java

```diff
@@ -314,6 +314,45 @@ public String getNicknameMention() {
                 .flatMap(ignored -> Mono.just(PermissionSet.all()))
                 .switchIfEmpty(Mono.zip(getEveryonePerms, getRolePerms, PermissionUtil::computeBasePermissions));
     }
+    
+    /**
+	 * Requests to determine if this member is higher in the role hierarchy than member.
```

Comment:
I like to add "provided" or "supplied" to signify I'm talking about the parameter.
`Requests to determine if this member is higher in the role hierarchy than the provided member.`